site-pages/wishlist/wishlist.css

```diff
@@ -0,0 +1,56 @@
+.wishlist-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 40px;
+    margin: 2.5rem 1rem;
+}
+.main-info-cont, 
+.sub-info-cont,
+.footer{
+    text-align: center;
+}
+.wishlist-head{
+    font-weight: 500;
+}
+.wishlist-cards{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
short-hand property can be used as _flex-flow: row wrap;_

Reply:
Okay, will change it...